recipes/singledispatchmethod/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "singledispatchmethod" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/singledispatchmethod-{{ version }}.tar.gz
+  sha256: 183a7fbeab53b9c9d182f8b8f9c2d7e109a7d40afaa30261d81dd8de68cd73bf
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools-scm
+  run:
+    - python
+    - singledispatch  # [py<34]
```

Comment:
Conda-forge only supports `Python >=3.7` for non `noarch` packages so you can remove that and make the minimum python version 3.4 and make this package `noarch`.

Reply:
OK sounds good. By "remove that", do you mean remove the line `- singledispatch  # [py<34]` or just the `# [py<34]`? Sorry, newb contributor here!

Reply:
Yes sorry should've been more clear. 
```suggestion
```

Basically, when a package has a version selector on a dependency it needs to be built for every python version separately. Since conda-forge only builds for python3.7, 3.8, 3.9 and 3.10, you'll never hit python3.3 and need this package.

When a package is noarch, it can be built once and installed under every python version that is allowed in your dependencies. 
Since Python3.3 requires this package, your minimum version should therefore be anything greater than that.

_Note_: I haven't checked if upstream has declared a higher python boundary than that anywhere else yet.

Reply:
Great, thanks for the clarification.